docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -107,15 +107,25 @@ is used.
       * ``ZSTD``
       * ``GZIP``
     - ``ZSTD``
+  * - ``iceberg.use-file-size-from-metadata``
+    - Some Iceberg writers populate incorrect file sizes in the metadata. When
```

Comment:
Is there an Iceberg GitHub issue or something we can link for when specifically this happens?

Reply:
https://github.com/apache/iceberg/issues/1980

Looks like this one

Reply:
I don't know how you found it..

Reply:
good idea, let's add this